src/statsd_client.cpp

```diff
@@ -0,0 +1,268 @@
+/**
+Copyright (c) 2014, Rex
```

Comment:
Is this GPL? Personally I'd rather not use this at all than have this code if this is the case.

Reply:
Nope, the repo claims to be MIT. Although it appears that the statement below is actually the 3-clause BSD :thinking: 
https://github.com/jlopp/statoshi#license
>Statoshi is released under the terms of the MIT license. See COPYING for more information or see http://opensource.org/licenses/MIT.